src/scripts/controller.js

```diff
@@ -1,5 +1,6 @@
 window.onload = function() {
 
   coursesAdder(".courses", coursesList);
+  certificatesAdder(".certificate-list", certificatesList);
```

Comment:
Your `certificatesAdder` will never fire.
Please check your console before committing. I guess you missed the error.

Reply:
Also I don't see `.certificate-list` in your pug